go/consensus/api/transaction/v0/envelope.go

```diff
@@ -105,7 +112,7 @@ type Envelope struct {
 
 // Open opens an envelope, and returns the payload after verifying the
 // Account and signatures.
-func (e *Envelope) Open(dst interface{}) error {
+func (e *Envelope) Open(context signature.Context, dst interface{}) error {
```

Comment:
If the multisig envelope is now something general (since it seems it supports different contexts as it is used for entities), should it be moved somewhere outside the consensus API (e.g., somewhere under common/crypto)?

Reply:
Yeah that's probably a good idea.  The construct itself is generic to a point.  There is a caveat that things that care about replay prevention (eg: actual transactions) need to include protection in the payload since it's trivial to re-submit an envelope with different numbers of signatures.

Reply:
I'll move this once I'm not in the middle of working on another part of this.